modules/database/sqlite/activate.php

```diff
@@ -148,7 +148,6 @@ function( $redirect_location ) {
 			// Use direct database query for Performance Lab modules to
 			// prevent module activation logic from firing again.
 			update_option( 'active_plugins', $active_plugins_option );
-			global $wpdb;
```

Comment:
@aristath Seeing this commit and wondering, is that really a duplicate? Have you tried running that code again, does it still work? The only other `$wpdb` I see here in this closure is in line 64, however that is in an if clause, so it wouldn't be relevant for this logic path.

Reply:
Yes, I reran the code, and did not encounter any issues.
However, you are correct, the 1st one was inside a condition so I reverted the commit 👍